src/web/browser/ga/ga.ts

```diff
@@ -174,3 +174,15 @@ export const sendPageView = (): void => {
 
     trackLCP(send);
 };
+
+export const trackNonClickInteraction = (actionName: string): void => {
+    const { ga } = window;
+
+    if (ga) {
+        const send = `${tracker.name}.send`;
+
+        ga(send, 'event', 'Interaction', actionName, {
+            nonInteraction: true, // to avoid affecting bounce rate
```

Comment:
> send event `Interaction`, non interaction = true
doesn't sound right

Reply:
Could be worth adding in a bit more detail about what exactly we're doing here, eg a link to https://support.google.com/analytics/answer/1033068#NonInteractionEvents or a similar explanation

Reply:
Good points. Honestly, I copied the long-standing (4+ years) implementation of `trackNonClickInteraction`, including comments that we've been using on `frontend`: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/google.js#L13, so unfortunately I don't have the detail behind why we've done this in the past.

Reply:
wasn't clear it was just a copy! good old github.  But that link is useful Imogen!

Reply:
I've updated the comment, adding the link @i-hardy provided.